stdlib/Dates/src/conversions.jl

```diff
@@ -3,7 +3,7 @@
 # Conversion/Promotion
 
 """
-    Date(dt::DateTime) -> Date
+    Date(dt::DateTime)::Date
```

Comment:
Should cases like this be annotated at all, or is it just clutter? 

The default assumption that `SomeType(...)` must construct a `SomeType`, exceptions (at there any?) need a giant warning. Annotating all the normal cases, if anything, seems to hint that there might be exceptions to watch out for.

Reply:
Yeah, I agree with you: Just clutter.